src/index.js

```diff
@@ -1 +1,5 @@
 import './style.css';
+import headerLogoImg from '../assets/main_logo.png';
+
+const headerLogo = document.getElementById('header-logo');
+headerLogo.src = headerLogoImg;
```

Comment:
We don't need all of this javascript just to show an image. We should use src to put the logo. It's easier and better readable code then 3 lines of js.
```html
   <div>
        <img id="header-logo" src="../assets/main_logo.png" alt="logo" />
    </div>
```

Reply:
I understand what you mean, the way the Webpack is configured to use images is through file and URL loaders, because of this you can't use the src attribute directly to the element.
I couldn't find a solution to this before, if you know a solution for this, or maybe we can check together, maybe you can help me fix it

Reply:
### I found the problem 🐛 
In the html you have to use the line `src="./assets/main_logo.png"`, and before you run your project, run the script `npm run build` so webpack bundles the images into the `"./dist/assets"` directory

Reply:
> ### I found the problem 🐛
> In the html you have to use the line `src="./assets/main_logo.png"`, and before you run your project, run the script `npm run build` so webpack bundles the images into the `"./dist/assets"` directory

I was wrong, this wasn't enough, I had to install another plugin that copies files, in this case it is configured in the `webpack.configure.js` that it takes the `root > assets` directory and duplicates it into the `dist` directory.
Each time you want to see your project you have to run `npm run build` to make sure that the assets folder is duplicated